solidity/ecdsa/test/fixtures/index.ts

```diff
@@ -40,6 +40,7 @@ export const params = {
   dkgResultChallengePeriodLength: 10,
   dkgResultSubmissionTimeout: 30,
   dkgSubmitterPrecedencePeriodLength: 5,
+  sortitionPoolRewardsBanDuration: 604800, // 7 days
```

Comment:
I take that this value is for testing purposes only and doen't have to match what we have in the WalletRegistry contract 14 days?

Reply:
I do not think we have finalized this value. But yeah, it's probably better to test with the same value that is used in the constructor. See https://github.com/keep-network/keep-core/pull/2868/commits/f87b1f440905744c17066a32aec8845c350f88f2, please.